sunpy/net/dataretriever/tests/test_swap.py

```diff
@@ -25,13 +25,14 @@ def test_get_url_for_time_range(timerange, url_start, url_end, level):
     assert urls[0] == url_start
     assert urls[-1] == url_end
 
+trange = Time('2015/12/30 00:00:00','2015/12/31 00:05:00')
 def test_can_handle_query():
-    assert swap.SWAPClient._can_handle_query(Time('2015/12/30 00:00:00','2015/12/31 00:05:00'), Instrument('swap'), a.Level(1)) is True
-    assert swap.SWAPClient._can_handle_query(Time('2015/12/30 00:00:00','2015/12/31 00:05:00'), Instrument('swap'), a.Level('q')) is True
-    assert swap.SWAPClient._can_handle_query(Time('2015/12/30 00:00:00','2015/12/31 00:05:00'), Instrument('swap'), a.Level('s')) is False
-    assert swap.SWAPClient._can_handle_query(Time('2015/12/30 00:00:00','2015/12/31 00:05:00'), Instrument('swap')) is False
-    assert swap.SWAPClient._can_handle_query(Time('2015/12/30','2015/12/31')) is False
-    assert swap.SWAPClient._can_handle_query(Time('2015/12/30 00:00:00','2015/12/31 00:05:00'), Instrument('eve')) is False
+    assert swap.SWAPClient._can_handle_query(trange, Instrument('swap'), a.Level(1))
+    assert swap.SWAPClient._can_handle_query(trange, Instrument('swap'), a.Level('q'))
+    assert swap.SWAPClient._can_handle_query(trange, Instrument('swap'), a.Level('s'))
+    assert not swap.SWAPClient._can_handle_query(trange, Instrument('swap'))
+    assert not swap.SWAPClient._can_handle_query(trange) is False
```

Comment:
Please @sudk1896 pay more attention when updating changes. This one would break as well as you are saying:
```python
assert not False is False
```